tests/test_utils/tests.py

```diff
@@ -773,6 +778,10 @@ def test_count(self):
         dom2 = parse_html('<div><p>foo</p><p>bar</p></div>')
         self.assertEqual(dom2.count(dom1), 1)
 
+        # children are a superset of the tested html
```

Comment:
I'm finding this comment too opaque. Can I ask you to rephrase so it guides a fresh reader quickly/easily?